cumulus_library/cli.py

```diff
@@ -89,7 +89,7 @@ def reset_data_path(self, study: PosixPath) -> None:
 
     ### Creating studies
 
-    def clean_study(self, targets: List[str]) -> None:
+    def clean_study(self, targets: List[str], explicit_prefix=False) -> None:
```

Comment:
Can you expand a bit on what the explicit prefix stuff is all about? It makes me lightly nervous to allow cleaning on any old prefix - just seems very "shoot yourself in the foot" territory.

If it's to clean up for oddball historical tables... Maybe instead of an explicit prefix, we could do explicit full-name matching. I.e. Don't allow cleaning on `^m*` but rather just `^mistake$` -- or is that not a real issue / not a good idea?

Reply:
My concern was just avoiding accidental deletion.

- Yeah you could manually drop the tables in Athena, but that's more technical - I had assumed this was exactly that, but for casuals who aren't 100% how to do that.
  - Whether this "full name" idea makes sense depends on how many tables we're talking about in those use cases. Like 10 might be annoying to do individually. But 3? :shrug:
- It might be nice to have like, a guard against destructive actions, since this can affect tables the user might not be thinking about (like, they forget that covid_symptoms__ exists when they're trying to delete some old covid_junk)
  - Something like `This command will delete these tables: [...] Continue? [y/N]`
  - Doesn't have to be this PR, but it feels like a nice guard, now that we're moving past single study actions.

Regardless, this is fine - I just get real nervous about wildcard deletes with no ability to undo it :smile:

Reply:
(I know that's ironic coming from someone who recently had the NdjsonFormat class uncritically delete a dir. But that did get fixed for a reason.)

Reply:
i like the list & confirm idea - i'll either squeeze it in now or make a ticket for it.

Reply:
#129